File: JavaScript/leetcode75/src/test/Problem73_MinimumNumberOfArrowsToBurstBalloons.test.js
const findMinArrowShots = require('../main/Problem73_MinimumNumberOfArrowsToBurstBalloons');

test('should return min arrows', () => {
  const points = [
    [10, 16],
    [2, 8],
    [1, 6],
    [7, 12],
  ];

  expect(findMinArrowShots(points)).toBe(2);
});

File: Java/leetcode75/src/test/java/org/phoenix/leetcode/challenges/Problem73_MinimumNumberOfArrowsToBurstBalloonsTest.java
package org.phoenix.leetcode.challenges;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class Problem73_MinimumNumberOfArrowsToBurstBalloonsTest {

    private final Problem73_MinimumNumberOfArrowsToBurstBalloons test = new Problem73_MinimumNumberOfArrowsToBurstBalloons();

    @Test
    void findMinArrowShots() {
        int[][] points = new int[][]{{10, 16}, {2, 8}, {1, 6}, {7, 12}};

        assertEquals(2, test.findMinArrowShots(points));
    }
}